radixdlt-java/src/main/java/com/radixdlt/client/atommodel/tokens/TokenDefinitionParticle.java

```diff
@@ -43,9 +43,9 @@
 	@DsonOutput(Output.ALL)
 	private UInt256 granularity;
 
-	@JsonProperty("icon")
+	@JsonProperty("icon_url")
```

Comment:
We are mixing snake and camelCase a lot, but mostly we do camelCase (I think?), can we maybe decide to stick to always be camelCasing our JSON Keys? :) 

**If** there is a **complete tie** in opinions between camelCase and snake_case, we should go for camelCase since it saves A LOT of code in the Swift world, talking about a couple of hundreds of lines of code, maybe more...

Reply:
I'll change it to camelCase.  That seems to be the general consensus anyway from other comments.